### PR TITLE
Fix issue with include_fec attribute not being set

### PR DESCRIPTION
**Pull request by quartercastle**
### Description

The `include_fec` attribute in `fec_output_settings` wasn't set correctly.

### Relations

Relates #27824

### Output from Acceptance Testing

```
$ make testacc TESTS=TestAccMediaLiveChannel_udp_output_settings PKG=medialive
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/medialive/... -v -count 1 -parallel 20 -run='TestAccMediaLiveChannel_udp_output_settings'  -timeout 180m
=== RUN   TestAccMediaLiveChannel_udp_output_settings
=== PAUSE TestAccMediaLiveChannel_udp_output_settings
=== CONT  TestAccMediaLiveChannel_udp_output_settings
--- PASS: TestAccMediaLiveChannel_udp_output_settings (83.58s)
PASS
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/medialive	85.575s
...
```

### CODE REVIEW

**Comment by quartercastle** [comment]
> Hey @quartercastle 👋 Thank you very much for your contribution! At times, our maintainers need to make direct edits to pull requests in order to help get it ready to be merged. Your current settings do not allow maintainers to make such edits. To help facilitate this, update your pull request to allow such edits as described in GitHub's [Allowing changes to a pull request branch created from a fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork) documentation. (If you're using a fork owned by an organization, your organization may not allow you to change this setting. If that is the case, let us know.)

The organisation i'm contributing from doesn't allow this.

File: internal/service/medialive/channel_test.go [hunk 1]
```diff
@@ -142,6 +142,68 @@ func TestAccMediaLiveChannel_m2ts_settings(t *testing.T) {
 	})
 }
 
+func TestAccMediaLiveChannel_udp_output_settings(t *testing.T) {
```
[the remark this answers is not in the document]
Comment (quartercastle) [reply internal/service/medialive/channel_test.go]: Yes, i changed it in updated it in bc0e72e. Thanks for the suggestion!

**Review by johnsonaj — APPROVED** [review]
LGTM 🚀 

```console
$ make testacc TESTS=TestAccMediaLiveChannel_ PKG=medialive
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/medialive/... -v -count 1 -parallel 20 -run='TestAccMediaLiveChannel_'  -timeout 180m
--- PASS: TestAccMediaLiveChannel_audioDescriptions_codecSettings (83.22s)
--- PASS: TestAccMediaLiveChannel_hls (88.04s)
--- PASS: TestAccMediaLiveChannel_disappears (97.24s)
--- PASS: TestAccMediaLiveChannel_basic (98.69s)
--- PASS: TestAccMediaLiveChannel_update (117.86s)
--- PASS: TestAccMediaLiveChannel_UDP_outputSettings (168.15s)
--- PASS: TestAccMediaLiveChannel_m2ts_settings (173.71s)
--- PASS: TestAccMediaLiveChannel_updateTags (173.87s)
--- PASS: TestAccMediaLiveChannel_status (221.26s)
PASS
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/medialive	224.210s
```

**Comment by johnsonaj** [comment]
@quartercastle thanks for the contribution! 🎉

**Pull request merged**